### PR TITLE
WRN-17310: Add sampler for Drawing in enact/packages/sampler

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added sampler for ui/Drawing

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-17310

### CODE REVIEW

File: packages/sampler/stories/default/Drawing.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {boolean, range, select} from '@enact/storybook-utils/addons/controls';
+
+import Drawing from '../../../ui/Drawing';
```
Comment (stanca-pop-lgp) [inline packages/sampler/stories/default/Drawing.js:4]: Could you import this from the enact package, not locally?

File: packages/sampler/stories/default/Drawing.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+const DrawingConfig = mergeComponentMetadata('Drawing', Drawing);
+
+export default {
+	title: 'UI/Drawing',
```
Comment (stanca-pop-lgp) [inline packages/sampler/stories/default/Drawing.js:9]: Please sort.

Comment (adrian-cocoara-lgp) [reply packages/sampler/stories/default/Drawing.js:9]: this is the order used in every sample. we should leave this as it is now

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**